logstash/datadog_checks/logstash/__about__.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.0'
+__version__ = '1.0.1'
```

Comment:
This is an enhancement so should be a minor upgrade in semver.
```suggestion
__version__ = '1.1.0'
```